h5netcdf/tests/test_h5netcdf.py

```diff
@@ -11,10 +14,35 @@
 
 from pytest import raises
 
+try:
+    import h5pyd
+    without_h5pyd = False
+except ImportError:
+    without_h5pyd = True
+
+
+@pytest.fixture(params=['testfile.nc', 'hdf5://testfile'])
+def tmp_netcdf(request, tmpdir):
+    if request.param.startswith(('http:', 'hdf5:')):
```

Comment:
please put these prefixes into a module level constant that you can re-use throughout